app/views/protocols/view_details/_project_fields.html.haml

```diff
@@ -39,7 +39,7 @@
       %label.col-lg-4
         = t(:protocols)[:projects][:information][:funding_status]
       .col-lg-8
-        = protocol.funding_status
+        = protocol.funding_status.nil? ? '' : protocol.funding_status.titleize
```

Comment:
`protocol.funding_status.try(&:titleize)` should do the same thing but a little simpler